src/components/Header/Header.jsx

```diff
@@ -1,6 +1,14 @@
-// import the React library here
-// import a css file containig article styles
+import React from 'react';
+import './Header.css';
+import Navigation from './Navigation/Navigation';
```

Comment:
```suggestion
import Navigation from '../Navigation/Navigation';
```

I guess it should fix failed tests